test/index.js

```diff
@@ -136,6 +136,8 @@ describe('External API', () => {
       const wrongAppInfo = Object.assign({}, appInfo, { customExecPath: 'non-array-exec-path' });
       return should(initialiseApp(wrongAppInfo)).be.rejectedWith('Exec command must be an array of string arguments');
     });
+
+    it('throws error for appInfo property', () => should(initialiseApp({ id: 'id1', vendor: 'vendor' })).be.rejected());
```

Comment:
Perhaps better to rename the test to 'throws errors for appInfo missing property' and use `rejectedWith`?